next16.1.1 turbopack Build Error: Module not found: Can't resolve

> 提示：越详细的信息越有助于排查和解决问题，如果方便请加入本项目 README.md 最下面的 QQ 群或者微信群提供更详细的信息

## 插件版本

_请提供使用的 code-inspector-plugin 版本号……_
code-inspector-plugin: 1.3.4
next: 16.1.1
turbo: 2.6.1
react: 19.2.3
## Bug 类型

- [ ] 安装插件后启动项目编译失败

## Bug 描述
每次重启 dev 命令会报错Module not found: Can't resolve 'code-inspector-plugin/dist/append-code-5678.js'，需要删除.next 文件夹才能启动成功

_请描述一下你的 bug，最好附上一张报错截图……_

<img width="1190" height="372" alt="Image" src="https://github.com/user-attachments/assets/42c225d3-b1b9-4919-b9fd-ddcbb019d521" />

## 系统

- [ ] mac

## 你使用的打包器及版本

- [ ] turbopack

## 你使用的 web 框架

- [ ] next.js

## 自检信息

- [ ] 你的浏览器、IDE、代码是否在一台机器上(非远程开发机或者云开发机情况)
- [ ] 浏览器控制台是否有打印组合按键提示信息<br />
      <img src="https://github.com/zh-lx/code-inspector/assets/73059627/77bcef30-88a5-4f58-b306-a92e01ecef8f" width="500" />

- [ ] 页面 DOM 是否有注入 `data-insp-path` 属性<br />
      <img src="https://github.com/zh-lx/code-inspector/assets/73059627/0523f9fb-e755-4561-9284-8970e4081bcc" width="600" />

--- a/packages/core/src/server/use-client.ts
+++ b/packages/core/src/server/use-client.ts
@@ -98,7 +98,7 @@ function addNextEmptyElementToEntry(content: string) {
   return s.toString();
 }
 
-function addImportToEntry(content: string, webComponentNpmPath: string) {
+function addImportToEntry(content: string, webComponentFilePath: string) {
   let hasAddedImport = false;
   const s = new MagicString(content);
 
@@ -125,7 +125,7 @@ function addImportToEntry(content: string, webComponentNpmPath: string) {
       ) {
         s.prependRight(
           node.end,
-          `;import ${NextEmptyElementName} from '${webComponentNpmPath}';`
+          `;import ${NextEmptyElementName} from '${webComponentFilePath}';`
         );
         hasAddedImport = true;
       }
@@ -135,7 +135,7 @@ function addImportToEntry(content: string, webComponentNpmPath: string) {
   if (hasAddedImport) {
     return s.toString();
   } else {
-    return `import ${NextEmptyElementName} from '${webComponentNpmPath}';${s.toString()}`;
+    return `import ${NextEmptyElementName} from '${webComponentFilePath}';${s.toString()}`;
   }
 }
 
@@ -344,17 +344,21 @@ export async function getCodeWithWebComponent({
       hasWritePermission(record.output)
     ) {
       writeEslintRcFile(record.output);
-      const webComponentNpmPath = writeWebComponentFile(
+      const webComponentFilePath = writeWebComponentFile(
         record.output,
         injectCode,
         getProjectRecord(record)?.port || 0
       );
-      if (!file.match(webComponentNpmPath)) {
+      if (!file.match(webComponentFilePath)) {
+        const relativePath = path.relative(
+          path.dirname(file),
+          webComponentFilePath
+        );
         if (isNextjs) {
-          code = addImportToEntry(code, webComponentNpmPath);
+          code = addImportToEntry(code, relativePath);
           code = addNextEmptyElementToEntry(code);
         } else {
-          code = `import '${webComponentNpmPath}';${code}`;
+          code = `import '${relativePath}';${code}`;
         }
       }
     } else {
@@ -385,10 +389,9 @@ function writeWebComponentFile(
   port: number
 ) {
   const webComponentFileName = `append-code-${port}.js`;
-  const webComponentNpmPath = `code-inspector-plugin/dist/${webComponentFileName}`;
   const webComponentFilePath = path.resolve(targetPath, webComponentFileName);
   fs.writeFileSync(webComponentFilePath, content, 'utf-8');
-  return webComponentNpmPath;
+  return webComponentFilePath;
 }
 
 export function isNextjsProject() {

